plugins/janus_audiobridge.c

```diff
@@ -2757,12 +2767,12 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 			int error = 0;
 			OpusDecoder *decoder = opus_decoder_create(participant->room->sampling_rate, 1, &error);
 			if(error != OPUS_OK) {
-				JANUS_LOG(LOG_ERR, "Error resetting Opus decoder...\n");
+				JANUS_LOG(LOG_ERR, "(participant %"SCNu64") Error resetting Opus decoder...\n", participant->user_id);
```

Comment:
Unless they're just there for temporary debugging purposes, please remove all those `(participant %"SCNu64")` prefixes you added. I guess in some places it might be useful to have, but this format is not in line with how we prefix such things normally. I can take care of that as part of a different commit.

Reply:
remove this information but I can tell you that in some traces it's important to have participant info.